doc/source/whatsnew/v0.21.0.txt

```diff
@@ -127,7 +127,7 @@ Sparse
 Reshaping
 ^^^^^^^^^
 
-
+- `argmin`, `argmax`, `idxmin`, and `idxmax` on Series, DataFrame, and GroupBy objects work correctly with floating point data that contains infinite values (:issue:`13595`). These functions now also work with string data, as long as there are no missing values.
```

Comment:
use double-backticks around the function names ``argmin``, as well for ``Series`` and ``DataFrame``. you can say with groupby as ``GroupBy`` is not an external term.